ext/Pod-Html/t/pod2html-lib.pl

```diff
@@ -41,20 +45,40 @@ sub convert_n_test {
     my $infile   = catpath $vol, $new_dir, "$podfile.pod";
     my $outfile  = catpath $vol, $new_dir, "$podfile.html";
 
-    my %default_args_table = (
+    my %args_table = (
         infile      =>    $infile,
         outfile     =>    $outfile,
         podpath     =>    't',
         htmlroot    =>    '/',
         podroot     =>    $cwd,
     );
-    my @default_args_list = map { "--${_}=" . $default_args_table{$_} }
-        keys %default_args_table;
-    # To add/modify args to p2h, use @p2h_args
-    Pod::Html::pod2html(
-        @default_args_list,
-        @p2h_args,
-    );
+    my %no_arg_switches = map { substr($_,2) => 1 } ( qw|
+           --flush --recurse --norecurse
+           --quiet --noquiet --verbose --noverbose
+           --index --noindex --backlink --nobacklink
+           --header --noheader --poderrors --nopoderrors
+    | );
+    if (defined $p2h_args_ref) {
+        for my $sw (keys %{$p2h_args_ref}) {
```

Comment:
If I was supreme monarch, we would use postfix deref for all non-dual-life code.  But I am willing to admit that I Am not…

Reply:
> If I was supreme monarch, we would use postfix deref for all non-dual-life code. But I am willing to admit that I am not…

As has so often been the case, because the most advanced version of `perl` that I was permitted to use (while employed) or chose to use (in CPAN modules) was only 5.14, I've never used or learned postfix deref.  :-(